Gruntfile.js

```diff
@@ -188,7 +185,19 @@ module.exports = function (grunt) {
     // Exorcise, Uglify, Concat are used to create a minimized freedom.js with
     // correct sourcemap. Uglify needs an explicit 'sourceMapIn' argument,
     // requiring that exorcise be used before hand. Concat is able to properly
-    // attach a banner while maintaining the correct source-map offsets.
+    // attach a banner while maintaining the correct source-map offsets.    
+    // The replace tasks removes a charset declaration that breaks concat -
+    // it should be removed once concat copes better.
+    replace: {
```

Comment:
Loading up the generated freedom.min.js(.map) in Chrome and playing with it a bit, it looks like line numbers correspond appropriately. Not sure if you were envisioning/intending some other check, let me know - thanks!